app/controllers/feature_flags_controller.rb

```diff
@@ -1,21 +1,3 @@
 class FeatureFlagsController < ApplicationController
-  before_action :check_flags_overrideable, only: %i[edit update]
-
   def index; end
-
-  def edit
-    @model = FeatureFlagOverride.find_by(key: params[:id]) || FeatureFlagOverride.new(key: params[:id])
-  end
-
-  def update
-    model = FeatureFlagOverride.find_or_create_by!(key: params[:id])
-    model.update!(params.require(:feature_flag_override).permit(:value))
-    redirect_to feature_flags_path
-  end
-
-private
-
-  def check_flags_overrideable
-    redirect_to root_path unless FeatureFlags.overrideable?
-  end
```

Comment:
So we're getting rid of this user interface we created for creating 'FeatureFlagOverride' database records, because now we have a new and shiny user interface for creating FeatureFlagOverride records provided by RailsAdmin.